manifests/init.pp

```diff
@@ -47,8 +47,8 @@
   Boolean $enc_upload_facts = true,
   Enum['yaml', 'json'] $enc_fact_extension = $puppetserver_foreman::params::enc_fact_extension,
   Stdlib::Absolutepath $puppet_home = $puppetserver_foreman::params::puppet_home,
-  String $puppet_user = 'puppet',
-  String $puppet_group = 'puppet',
+  String $puppet_user = $puppetserver_foreman::params::puppet_user,
+  String $puppet_group = $puppet_user,
```

Comment:
let me know what you think about it. I think in 99% of the situations, user and group are identical. In case someone want's to run on a customer user/group, it's now enough to just `puppet_user`. I think that's a nice convenience feature.